app/views/super_admin/content.py

```diff
@@ -47,6 +48,7 @@ def index_view():
         for key, value in dic.items():
             settings[key] = value[0]
             set_settings(**settings)
+        flash("Changes Saved Successfully!!!")
```

Comment:
Please drop the exclamation marks. A simple `Changes have been saved.` would do.